src/lib/WIFI/devWIFI.cpp

```diff
@@ -731,6 +731,8 @@ static void startMDNS()
     MDNSResponder::hMDNSService service = MDNS.addService(instance.c_str(), "http", "tcp", 80);
     MDNS.addServiceTxt(service, "vendor", "elrs");
     MDNS.addServiceTxt(service, "target", (const char *)&target_name[4]);
+    MDNS.addServiceTxt(service, "device", (const char *)&device_name);
+    MDNS.addServiceTxt(service, "product", (const char *)&product_name);
```

Comment:
I think it is clearer to write these as `(const char *)device_name` because the array will automatically decay to a "pointer to the first element of X" and the address operator returns "pointer to an array of size X" which can get you in a lot of trouble (not that it does here, but convention is to use automatic decay).
![image](https://user-images.githubusercontent.com/677183/180470239-c1dd6453-2710-4595-b1af-0776094d69cc.png)